internal/proposals/unified-approach-to-logging-levels.md

```diff
@@ -0,0 +1,62 @@
+# Unified approach to logging levels
+
+It must be possible to easily set a more verbose logging level of each component within the Kyma ecosystem while troubleshooting.
+
+We also have some "external" components (not implemented by the Kyma team) and we need an approach on how to change the logging level in them. For debugging reasons the level could be changed without any code and charts changes just with the in-cluster resource change.
+
+## What to put into each severity level
+
+As we decided to use Zap library as the logger I'll describe all the needed levels from this particular one:
+
+- FATAL - a component setup failed and need to be restarted,
```

Comment:
I will describe it this way: `component reached an catastrophic error and cannot go further`

Reply:
Done!